### PR TITLE
Nayem ddb query implementation

### PR DESCRIPTION
**Pull request by lucifer7663**
Fixes #

- [x] Does not depend on other PRs
- [ ] Depends on other PR being merged first: #

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [x] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?
- implemented ddb user activity query
## How to test

- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

## Reminder list

- Determine whether REST API Documentation should be updated
- Make an assessment on whether unit tests should be updated and/or added
- Ensure follow up tasks are linked to this PR

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

File: src/main/java/com/serverless/Handler.java [hunk 1]
```diff
@@ -29,19 +36,194 @@
...
+                        List<Object> data = new ArrayList<>();
+                        Item item = null;
+                        while (iterator.hasNext()) {
+                            item = iterator.next();
+                            data.add(item.toString());
+                        }
+                        return ApiGatewayResponse.builder()
+                                .setObjectBody(data)
+                                .build();
+                    }catch (Exception e){
```
Comment (mhazad) [inline src/main/java/com/serverless/Handler.java:118]: 😞 🤦‍♂️ Malformatted code!  
```java
}catch (Exception e){
```
Please check your PR again and fix all formatting issues.

**Review by mhazad — CHANGES_REQUESTED** [review]
You are repeating mistakes. At the first glance, I found a code formatting issue. Please double-check your PR and only request for review when you will think that it's good to go.  Please do not waste your and my time for your negligence.

File: serverless.yml [hunk 2]
```diff
@@ -5,15 +5,18 @@ frameworkVersion: '3'
...
   name: aws
   runtime: java11
+  timeout: 600
+  httpApi:
+    cors: true
   environment:
     BucketName: ${env:BUCKETNAME}
     QueueName: ${env:QUEUENAME}
     TableName: ${env:TABLENAME}
+    Table_Name: ${env:QUERYTABLENAME}
```
Comment (mhazad) [inline serverless.yml:15]: The code should follow a certain convention. Previous environment variables were declared as Pascal Case, but this one is defined as Pascal Snake Case.

File: serverless.yml [hunk 3]
```diff
@@ -5,15 +5,18 @@ frameworkVersion: '3'
...
     TopicName: ${env:TOPICNAME}
     DisplayName: ${env:DISPLAYNAME}
-
-
-
+    baseUrl: ${env:BASEURL}
```
Comment (mhazad) [inline serverless.yml:18]: The code should follow a certain convention. Previous environment variables were declared as Pascal Case, but this one is defined as camel case.

File: serverless.yml [hunk 4]
```diff
@@ -5,15 +5,18 @@ frameworkVersion: '3'
...
+    webhook: ${env:WEBHOOK}
```
Comment (mhazad) [inline serverless.yml:19]: The code should follow a certain convention. Previous environment variables were declared as Pascal Case, but this one is defined as lower case.

File: src/main/java/com/serverless/Handler.java [hunk 5]
```diff
@@ -29,19 +36,194 @@
...
+                        .build();
+            }
+        }
+        if (input.containsKey("routeKey") && input.get("routeKey").equals("GET /user-activity")
+                && input.get("queryStringParameters") != null && input.containsKey("queryStringParameters")) {
+
+            ObjectParsing objectParsing = new ObjectParsing();
+            Map<String, Object> params = objectParsing.convertObjectToMap(input.get("queryStringParameters"));
+
+            if (params.containsKey("customerName") && params.containsKey("username") && params.containsKey("queryType")) {
```
Comment (mhazad) [inline src/main/java/com/serverless/Handler.java:90]: The code should follow a certain convention. 
customerName is camelCase
username is lowercase
queryType is camelCase. 

username should be camelCase

**Review by mhazad — CHANGES_REQUESTED** [review]
The code is much better now.  I have given some more feedback that is related to using different conventions while defining various variables.  Please address these.

File: serverless.yml (hunk 3 above)
Comment (lucifer7663) [reply serverless.yml]: variable naming convention is reverted to original Pascal Case.

File: serverless.yml (hunk 2 above)
Comment (lucifer7663) [reply serverless.yml]: variable naming convention is reverted to original Pascal Case.

File: src/main/java/com/serverless/Handler.java (hunk 5 above)
Comment (lucifer7663) [reply src/main/java/com/serverless/Handler.java]: username is now in camleCase

**Review by mhazad — APPROVED** [review]
LGTM

**Pull request merged**